hist

```diff
@@ -1,29 +1,38 @@
-#!/bin/bash
+#! /usr/bin/env bash
```

Comment:
What's the argument for a space after a shebang? I've never seen this.

I realize on FreeBSD, bash is probably at `/usr/local/bin/bash`. Is `/usr/bin/env` more likely than `/bin/bash`?